### DISCUSSION TITLE
Components 50% faster

### DISCUSSION THREAD

**Discussion by Angelk90** [callstack/react-native-paper#4105] [General]
### Ask your Question

I was reading these posts on Twitter, where they say that it is possible to speed up components by 50%.

Link: https://twitter.com/natebirdman/status/1695511232298783079

<img width="624" alt="Screenshot 2023-10-01 alle 16 17 27" src="https://github.com/callstack/react-native-paper/assets/20476002/11c3c4ae-b4e0-420d-b93e-75785cda386f">

Link: https://twitter.com/FernandoTheRojo/status/1707762822015267219

<img width="618" alt="Screenshot 2023-10-01 alle 16 17 39" src="https://github.com/callstack/react-native-paper/assets/20476002/0c7bf9ff-d7a2-4e20-a5c5-93d0c5ff8bb1">

What do you think, is it possible?

**Comment by SoyDiego** [comment]
Very interesting. I don't know why is not relevant since you wrote this one month ago.
What do you think @lukewalczak?

**Reply by Angelk90** [reply]
@SoyDiego, @lukewalczak : I'm wondering the same thing after two years.
Any news?
